ejb3/src/main/java/org/jboss/as/ejb3/security/EJBSecurityViewConfigurator.java

```diff
@@ -145,9 +145,7 @@ public void configure(DeploymentPhaseContext context, ComponentConfiguration com
         }
 
         final boolean securityRequired = beanHasMethodLevelSecurityMetadata || ejbComponentDescription.hasBeanLevelSecurityMetadata();
-        if (securityRequired) {
-            ejbComponentDescription.setSecurityRequired(securityRequired);
-        }
+        ejbComponentDescription.setSecurityRequired(securityRequired);
```

Comment:
`setSecurityRequired` should only be called if `securityRequired` is `true` (see https://issues.redhat.com/browse/WFLY-12301).

Reply:
@fjuma Ok reverted